test/e2e/mcd_test.go

```diff
@@ -143,6 +145,106 @@ func TestMCDeployed(t *testing.T) {
 	}
 }
 
+func TestUpdateSSH(t *testing.T) {
+	cs := framework.NewClientSet("")
+
+	// create a dummy MC with an sshKey for user Core
+	mcName := fmt.Sprintf("sshkeys-worker-%s", uuid.NewUUID())
+	mcadd := &mcv1.MachineConfig{}
+	mcadd.ObjectMeta = metav1.ObjectMeta{
+		Name:   mcName,
+		Labels: mcLabelForWorkers(),
+	}
+	// create a new MC that adds a valid user & ssh keys
+	tempUser := ignv2_2types.PasswdUser{Name: "core", SSHAuthorizedKeys: []ignv2_2types.SSHAuthorizedKey{"1234_test", "abc_test"}}
```

Comment:
could you rewrite this to be:
```go
	tempUser := ignv2_2types.PasswdUser{
		Name: "core",
		SSHAuthorizedKeys: []ignv2_2types.SSHAuthorizedKey{
			"1234_test",
			"abc_test",
		},
	}
```
it's more readable and respect the 80 chars columns

Reply:
done!